addons/website_slides/controllers/main.py

```diff
@@ -938,7 +938,10 @@ def slide_channel_send_share_email(self, channel_id, emails):
 
     @http.route(['/slides/channel/subscribe'], type='json', auth='user', website=True)
     def slide_channel_subscribe(self, channel_id):
-        return request.env['slide.channel'].browse(channel_id).message_subscribe(partner_ids=[request.env.user.partner_id.id])
+        # Presentation Published subtype
+        subtype = request.env.ref("website_slides.mt_channel_slide_published", raise_if_not_found=False)
+        return request.env['slide.channel'].browse(channel_id).message_subscribe(
```

Comment:
Maybe if 'mt_channel_slide_published' does not exist, simply skip subscription ? Otherwise he will end up without any subtype, which does not really makes sense (having a hanging follower that won't be notified of anything).